terraform/providers.tf

```diff
@@ -1,3 +1,4 @@
 provider "azurerm" {
   features {}
+  skip_provider_registration = true
```

Comment:
Is it worth calling this out in the commit message as something that threw us initially and what this means?

Reply:
Yup okie ...